roles/common/files/sensu_plugins/check-cpu.rb

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/env /opt/sensu/embedded/bin/ruby
 #
```

Comment:
we should move these to templates so we can dynamically set this ... but IMO we should merge as is for now and create a task for templatizing.